functions/shortenUrl/main.js

```diff
@@ -0,0 +1,25 @@
+const Logger = require('firebase-functions/lib/logger');
+const isSameField = require('./helpers/isSameField');
+const getConfig = require('./services/getConfig');
+const updateSnapshotShortUrl = require('./services/updateSnapshotShortUrl');
+
+const shortenUrl = async (snapshot) => {
+    const { longUrlField, shortUrlField } = getConfig();
+    if (isSameField(longUrlField, shortUrlField)) {
+        Logger.error('ERROR_SHORTEN_URL', {
+            errorMessage: 'The `LONG_URL` and `SHORT_URL` field names must not be the same',
```

Comment:
here we should use the real name of the variables LONG_URL_FIELD_NAME and SHORT_URL_FIELD_NAME to avoid confusion